unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/BackpackEditorHUDV2/AvatarSlot/AvatarSlotComponentModel.cs

```diff
@@ -10,5 +10,6 @@ public record AvatarSlotComponentModel
         public string category;
         public bool isHidden;
         public string hiddenBy;
+        public bool supportColor;
```

Comment:
It is maybe personal but instead of supportColor I would rename it to something more clear like: allowsColorChange